### PR TITLE
[homework #6] - Js basics 1 by David Azuaje

### PR DESCRIPTION
**Pull request by dazuaje**
Hey Mike! I hope you are doing well.

Here is the assignment Js Basics 1. Let me know if I have to fix something

### CODE REVIEW

**Review by solvd-mmelnyk — COMMENTED** [review]
I left comments in this PRs. Please fix the problems.
In general, you should also fix the:
1. Please format your code with Pretties. I see strange formatting in this file
2. Please review using of `const` and `let`
3. If you copy some code from some resources, please update it to look like yours
4. Your code is your face. Please make it beautiful and readable

let me know if you questions!

File: homeworks/david-azuaje/06-js-basics-1/script.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+  
+  //Task 8) => https://www.codewars.com/kata/570cc83df616a85944001315/train/javascript
+  
+  function countWords(string) {
+    let stringToArray = string.split(" ");
+    let index = stringToArray.map((elem, index) => {
+      return index + 1;
+    });
+    return Number(index.length);
+  }
```
[the remark this answers is not in the document]
Comment (dazuaje) [reply homeworks/david-azuaje/06-js-basics-1/script.js:130]: I already fixed

File: homeworks/david-azuaje/06-js-basics-1/script.js [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  
+  function findOdd(arr) {
+    return arr.find((item, index) => arr.filter(el => el == item).length % 2)
+  }
+  console.log(findOdd([0,1,0,1,0]));
+  
+  //Task 10) => https://www.codewars.com/kata/5526fc09a1bbd946250002dc/train/javascript
+  
+  function findOutlier(integers){
+    let pair = integers.filter(num => num % 2 === 0 );
```
[the remark this answers is not in the document]
Comment (dazuaje) [reply homeworks/david-azuaje/06-js-basics-1/script.js:144]: Done

File: homeworks/david-azuaje/06-js-basics-1/script.js [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    let odd = integers.filter(num => num % 2 !== 0);
+    return (pair.length === 1) ? pair[0] : odd[0]
+  }
+  console.log(findOutlier([1,1,0,1,1]))
+  
+  //Task 11) => Do it
```
[the remark this answers is not in the document]
Comment (dazuaje) [reply homeworks/david-azuaje/06-js-basics-1/script.js]: Finished

File: homeworks/david-azuaje/06-js-basics-1/script.js [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+  
+  
+  // Task 12) https://www.codewars.com/kata/filter-the-number
+  const filterNumber = (string) => string.match(/[0-9]/g);
```
[the remark this answers is not in the document]
Comment (dazuaje) [reply homeworks/david-azuaje/06-js-basics-1/script.js]: It's working now

File: homeworks/david-azuaje/06-js-basics-1/script.js [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+    }
+  
+    return result;
+  }
+  console.log(wave('Hello'));
+  
+  
+  //Task 19) => https://www.codewars.com/kata/59d398bb86a6fdf100000031
+  
+  function stringBreakers(n, string){
```
[the remark this answers is not in the document]
Comment (dazuaje) [reply homeworks/david-azuaje/06-js-basics-1/script.js:220]: It's working now

**Review by solvd-mmelnyk — APPROVED** [review]

**Pull request merged**